### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by ttahm3d**
- Changed conflicting class name
- added simple navbar and its example

### CODE REVIEW

**Review by krishanu2911 — APPROVED** [review]
No changes Required 👍

File: docs/navbar/index.html [hunk 1]
```diff
@@ -86,6 +86,44 @@ <h2 class="i-logo">
...
+                </ul>
+                <ul class="nav-item">
+                  Blogs
+                </ul>
+              </div>
+            </nav>
+          </div>
+
+          <div class="code-example">
+            <pre><code class="html">
```
Comment (vaibhav-mahalle) [inline docs/navbar/index.html:116]: is it a good practice to name class as html element?

**Review by vaibhav-mahalle — COMMENTED** [review]
some doubts I have raised. everything else looks good.

File: docs/navbar/index.html (hunk 1 above)
Comment (ttahm3d) [reply docs/navbar/index.html:116]: thats a requirement of highlightjs

**Review by vaibhav-mahalle — APPROVED** [review]

**Pull request merged**